tests/ast/nodes/test_evaluate_binop_int.py

```diff
@@ -0,0 +1,134 @@
+from hypothesis import (
+    example,
+    given,
+    settings,
+    strategies as st,
+)
+import pytest
+
+from vyper import (
+    ast as vy_ast,
+)
+from vyper.exceptions import (
+    ZeroDivisionException,
+)
+
+st_int32 = st.integers(min_value=-(2 ** 32), max_value=2 ** 32)
+
+
+@pytest.mark.fuzzing
+@settings(max_examples=50, deadline=1000)
+@given(left=st_int32, right=st_int32)
+@pytest.mark.parametrize("op", "+-*/%")
```

Comment:
Add some corner cases
```suggestion
st_int32 = st.integers(min_value=-(2 ** 32), max_value=2 ** 32)


@pytest.mark.fuzzing
@settings(max_examples=50, deadline=1000)
@given(left=st_int32, right=st_int32)
@example(left=1, right=1)
@example(left=1, right=-1)
@example(left=-1, right=1)
@example(left=-1, right=-1)
@pytest.mark.parametrize("op", "+-*/%")
```